examples/test_context_based_restrictions_v1_examples.py

```diff
@@ -0,0 +1,417 @@
+                'value': '169.23.56.234',
+            }
+
+            zone = context_based_restrictions_service.replace_zone(
+                zone_id=TestContextBasedRestrictionsV1Examples.zone_id,
+                if_match=TestContextBasedRestrictionsV1Examples.zone_rev,
+                name='SDK TEST - an example of updated zone',
+                account_id=TestContextBasedRestrictionsV1Examples.test_account_id,
+                addresses=[address_model],
+                description='SDK TEST - this is an example of updated zone',
+            ).get_result()
+
+            print(json.dumps(zone, indent=2))
+
+            # end-replace_zone
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_list_available_serviceref_targets_example(self):
+        """
+        list_available_serviceref_targets request example
+        """
+        try:
+            print('\nlist_available_serviceref_targets() result:')
+            # begin-list_available_serviceref_targets
+
+            service_ref_target_list = context_based_restrictions_service.list_available_serviceref_targets().get_result()
+
+            print(json.dumps(service_ref_target_list, indent=2))
+
+            # end-list_available_serviceref_targets
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_create_rule_example(self):
+        """
+        create_rule request example
+        """
+        try:
+            print('\ncreate_rule() result:')
+            # begin-create_rule
+
+            rule_context_attribute_model = {
+                'name': 'networkZoneId',
+                'value': TestContextBasedRestrictionsV1Examples.zone_id,
+            }
+
+            # Construct a dict representation of a RuleContext model
+            rule_context_model = {
+                'attributes': [rule_context_attribute_model],
+            }
+
+            # Construct a dict representation of a ResourceAttribute model
+            account_id_resource_attribute_model = {
+                'name': 'accountId',
+                'value': TestContextBasedRestrictionsV1Examples.test_account_id,
+            }
+
+            service_name_resource_attribute_model = {
+                'name': 'serviceName',
+                'value': TestContextBasedRestrictionsV1Examples.service_name,
+            }
+
+            # Construct a dict representation of a Resource model
+            resource_model = {
+                'attributes': [account_id_resource_attribute_model, service_name_resource_attribute_model],
+            }
+
+            rule = context_based_restrictions_service.create_rule(
+                contexts=[rule_context_model],
+                resources=[resource_model],
+                description='SDK TEST - this is an example of rule'
+            ).get_result()
+
+            print(json.dumps(rule, indent=2))
+
+            # end-create_rule
```

Comment:
I'd like to ask you to remove the comments between `# end-create_rule` and `# begin-create_rule` lines. We don't want to display comments in the reference.